src/aeb_fate_generate_ops.erl

```diff
@@ -247,6 +247,12 @@ ops_defs() ->
     , { 'DBG_UNDEF',           16#b3,  false,    true,   true,    ?GAS(0),       [a, a],     dbg_undef,               {string, any},    none, "Debug Op: Undefine a variable. Args = {var_name, register}" }
     , { 'DBG_CONTRACT',        16#b4,  false,    true,   true,    ?GAS(0),          [a],  dbg_contract,                    {string},    none, "Debug Op: Name the current contract. Args: {contract_name}"}
 
+    , { 'BYTES_SPLIT_ANY',     16#b5,  false,    true,   true,   ?GAS(10), [a, a, a],     bytes_split_any,   {bytes, integer}, variant, "Arg0 := bytes_split_any(Arg1, Arg2), where a positive Arg2 is the length of the first chunk, and a negative Arg2 is the length of the second chunk. Returns None if byte array is not long enough."}
+    , { 'BYTES_SIZE',          16#b6,  false,    true,   true,   ?GAS(10),    [a, a],          bytes_size,            {bytes}, integer, "Arg0 := bytes_size(Arg1), returns the number of bytes in the byte array."}
+    , { 'BYTES_TO_FIXED_SIZE', 16#b7,  false,    true,   true,   ?GAS(10), [a, a, a], bytes_to_fixed_size,   {bytes, integer}, variant, "Arg0 := bytes_to_fixed_size(Arg1, Arg2), returns Some(Arg1) if byte_size(Arg1) == Arg2, None otherwise."}
```

Comment:
How is it better than returning a bool? You still need to provide Arg1, so there is no point in getting it back. Also the name of the instruction is not descriptive.

Reply:
You have something of type `bytes()` and want something of type `bytes(n)` then a boolean wont help you? The `Arg1` has type `bytes(Arg2)` - perhaps the explanation can be improved...

Reply:
Ah, so it's casting! Makes sense